frontend/src/clients/BoardClient.ts

```diff
@@ -20,9 +20,8 @@ export class BoardClient extends HttpClient {
 
   getVerifyBoard = async (params: getVerifyBoardParams) => {
     try {
-      const result = await this.axiosInstance
-        .get(`/boards/${params.type}`, { params: { ...params } })
-        .then((res) => res)
+      const boardType = params.type === 'Classic Jira' ? 'classic-jira' : params.type.toLowerCase()
```

Comment:
If Type name in Params is different with boardType, it's better to create a mapping.

Reply:
right, I'll change this in next PR